.circleci/config.yml

```diff
@@ -5,6 +5,12 @@ jobs:
         docker:
             - image: circleci/node:8
         steps:
+            - restore_cache: # If any cache exists, it'll way speedup these checkouts
+                keys:
+                    - source-v1-{{ .Branch }}-{{ .Revision }}
+                    - source-v1-{{ .Branch }}-
+                    - source-v1-
+
```

Comment:
Found that moving the project_workspace (creating mostly) was way more time consuming than just caching our git and restoring + checkout. also with the above setup, it'll work for a brand new branch, skipping the really time consuming 5gb clone